experimental/dds/tree2/src/feature-libraries/modular-schema/view.ts

```diff
@@ -157,7 +157,7 @@ export class ViewSchema {
  */
 export interface ITreeSchema extends NamedTreeSchema, Sourced {
 	readonly structFields: ReadonlyMap<FieldKey, IFieldSchema>;
-	readonly mapFields: IFieldSchema;
+	readonly mapFields?: IFieldSchema;
```

Comment:
Maybe the docs should go here and they can be inherited from TreeStoredSchema?

Reply:
these actually do get docs as is (at least working in intellisense).  They are inherited from the interface it extends, TreeStoredSchema. The fields here are just more restricted types, not new fields.

Reply:
Oh nice, I completely missed that this was extending something.